model/task_lifecycle_test.go

```diff
@@ -38,38 +38,81 @@ var (
 
 func TestSetActiveState(t *testing.T) {
 	Convey("With one task with no dependencies", t, func() {
-		require.NoError(t, db.ClearCollections(task.Collection, build.Collection, task.OldCollection, VersionCollection))
+		require.NoError(t, db.ClearCollections(task.Collection, build.Collection, task.OldCollection, VersionCollection, commitqueue.Collection))
```

Comment:
Can we also add cases for 1) checking for appropriate updates of associated build / version statuses and 2) an execution task updating its display task?

Reply:
added